lib/Conversation/styles.scss

```diff
@@ -20,7 +20,7 @@ $conversation-meta-text-color: $neutral-base !default;
 /* ==========================================================================
    Styles
    ========================================================================== */
-.conversation {
+div.conversation {
```

Comment:
Do we need a `div` here? If there is a clash elsewhere then lets resolve that by changing that instance of `.conversation`.

Reply:
It was clashing with mark tags that shared the same class, I've renamed those